src/NuGet.Core/NuGet.CommandLine.XPlat/external/docs.md

```diff
@@ -0,0 +1,372 @@
+
+
+## Options
+
+- **`--configfile`**
+
+  The NuGet configuration file. If specified, only the settings from this file will be used. If not specified, the hierarchy of configuration files from the current directory will be used. To learn more about NuGet configuration go to https://docs.microsoft.com/en-us/nuget/consume-packages/configuring-nuget-behavior.
+
+## Examples
+
+- Disable a source with name of `mySource`:
+
+  ```dotnetcli
+  dotnet nuget disable source mySource
+  ```
+
+## See also
+
+- [Package source sections in NuGet.config files](/nuget/reference/nuget-config-file#package-source-sections)
+
+- [sources command (nuget.exe)](/nuget/reference/cli-reference/cli-ref-sources)
+    
+---file:/dotnet/core/tools/dotnet-nuget-enable-source
+---
+title: dotnet nuget enable source command
+description: The `dotnet nuget enable source` command enables an existing source in your NuGet configuration files. 
+author: nugetClient
+ms.date: 03/02/2020
+---
+# dotnet nuget enable source
+
+**This article applies to:** ✔️ .NET Core 3.1.200 SDK and later versions
+
+## Name
+
+`dotnet nuget enable source` - Enable a NuGet source.
+
+## Synopsis
+
+```dotnetcli
+`dotnet nuget enable source NAME [--configfile]`
+`dotnet nuget enable source [-h|--help]`
+```
+
+## Description
+
+The `dotnet nuget enable source` command enables an existing source in your NuGet configuration files. 
+
+## Arguments
+
+- **`NAME`**
+
+  Name of the source.
+
+
+## Options
+
+- **`--configfile`**
+
+  The NuGet configuration file. If specified, only the settings from this file will be used. If not specified, the hierarchy of configuration files from the current directory will be used. To learn more about NuGet configuration go to https://docs.microsoft.com/en-us/nuget/consume-packages/configuring-nuget-behavior.
+
+## Examples
+
+- Enable a source with name of `mySource`:
+
+  ```dotnetcli
+  dotnet nuget enable source mySource
+  ```
+
+## See also
+
+- [Package source sections in NuGet.config files](/nuget/reference/nuget-config-file#package-source-sections)
+
+- [sources command (nuget.exe)](/nuget/reference/cli-reference/cli-ref-sources)
+    
+---file:/dotnet/core/tools/dotnet-nuget-list-source
+---
+title: dotnet nuget list source command
+description: The `dotnet nuget list source` command lists all existing sources from your NuGet configuration files. 
+author: nugetClient
+ms.date: 03/02/2020
+---
+# dotnet nuget list source
+
+**This article applies to:** ✔️ .NET Core 3.1.200 SDK and later versions
+
+## Name
+
+`dotnet nuget list source` - Lists all configured NuGet sources.
+
+## Synopsis
+
+```dotnetcli
+`dotnet nuget list source [--format] [--configfile]`
+`dotnet nuget list source [-h|--help]`
+```
+
+## Description
+
+The `dotnet nuget list source` command lists all existing sources from your NuGet configuration files. 
+
+## Options
+
+- **`--format`**
+
+  Applies to the list action. Accepts two values: Detailed (the default) and Short.
```

Comment:
What do you think of something like `The format of the list command output` to clarify this affects the output? Also can we wrap `Detailed` and `Short` in back ticks?

Reply:
fixed in next commit